lib/wcc/contentful.rb

```diff
@@ -58,11 +109,10 @@ def self.init!
     when :eager_sync
       store = configuration.sync_store
 
-      client.sync(initial: true).items.each do |item|
-        # TODO: enrich existing type data using Sync::Indexer
-        store.index(item.dig('sys', 'id'), item)
-      end
       WCC::Contentful::Model.store = store
+
+      @next_sync_token = store.find("sync:#{configuration.space}:token")
```

Comment:
this handles the diffing between syncs?

Reply:
This lets us resume where we left off, ex. if we are using Redis for our cache then on a dyno restart we don't have to do the initial sync - the cache still exists.